doc/source/whatsnew/v0.22.0.txt

```diff
@@ -338,7 +338,8 @@ Reshaping
 Numeric
 ^^^^^^^
 
--
+- Fixed ``std`` and ``var`` computations for timedelta arrays not returning results in timedelta units (:issue:`18880`)
```

Comment:
this is a fix for 0.22? or for 0.23? (IOW do you need to have this for nansum)?

Reply:
This was was specific to nanstd, so I think I can work around it. The skipna stuff affected all the ops I think. I'll see how hard that is to work around.

Reply:
yeah rather leave this alone for now (IOW do it in 0.23).

Reply:
rebase on master. can you move to 0.23 (docs were renamed), prob easiest to just check this file from master and past in new one